Mathlib/Algebra/Group/Units.lean

```diff
@@ -674,6 +674,11 @@ theorem isUnit_of_mul_eq_one [CommMonoid M] (a b : M) (h : a * b = 1) : IsUnit a
 #align is_unit_of_mul_eq_one isUnit_of_mul_eq_one
 #align is_add_unit_of_add_eq_zero isAddUnit_of_add_eq_zero
 
+@[to_additive]
+theorem isUnit_of_mul_eq_one' [CommMonoid M] (a b : M) (h : a * b = 1) : IsUnit b := by
```

Comment:
I'm hopping back and forth... on the one hand shorter names good, on the other hand primed-names less good...
```suggestion
theorem isUnit_of_mul_eq_one_right [CommMonoid M] (a b : M) (h : a * b = 1) : IsUnit b := by
```

Reply:
Agreed and done.